Frontend/ui-driver/src/Flow.purs

```diff
@@ -1343,7 +1343,7 @@ homeScreenFlow = do
               modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen { props { routeVisible = true } })
               _ <- pure $ removeMarker "ic_vehicle_side"
               _ <- pure $ removeAllPolylines ""
-              _ <- lift $ lift $ doAff do liftEffect $ drawRoute coor "LineString" "#323643" true "ny_ic_src_marker" "ny_ic_dest_marker" 9 "NORMAL" source destination (specialLocationConfig "" "")
+              _ <- lift $ lift $ doAff do liftEffect $ drawRoute coor "LineString" "#323643" true "ny_ic_src_marker" "ny_ic_dest_marker" 9 "NORMAL" source destination (specialLocationConfig "" "") true
```

Comment:
@S-PRAVINKUMAR is this animation required in driver side too ?

Reply:
no. changed to false